README.md

```diff
@@ -364,6 +364,31 @@ func ExampleDatabase_QueryHistoricState() {
 ```
 </details>
 
+# Performance 
+
+Carmen is the backbone of [Fantom Sonic](https://fantom.foundation/sonicPage) blockchain. 
+It is the key component that enables high performance in a disk space conservative manner. 
+
+Carmen was extensively tested as part of the Sonic client (i.e. a blockchain node). 
+Three main scenarios were tested:
+1. **Realistic traffic:** a mix of transactions that resemble realistic traffic, such as token transfers, token mints, and complex multi-step swaps.
+2. **Token swaps:** transactions calling the uniswap’s contract, which handles multi-step swaps between tokens, demonstrating what the next generation of DEXs can achieve.
+3. **Token transfers:** transactions processing only ERC-20 transfers, demonstrating usage for next-generation wallets, payment providers.
+
+The client could process thousands of transactions per second as depicted in the table 
+and further detailed in [a blog post](https://blog.fantom.foundation/3-incredible-performances-from-fantom-sonic-closed-testnet/).
+
+| Configuration | Speed Tx/s |
+| ------------- | ---------- |
+| Realistic       | 2.000 |
+| Token swaps     | 4.000 |
+| Token transfers | 10.000 |
```

Comment:
I would remove the dot because in USA they use dot instead of comma to determine end of full number. Mby swap it for `_`?